docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -1,39 +1,66 @@
 ---
-title: "CNCF Buildpacks"
+title: "Cloud Native Buildpacks"
 linkTitle: "Buildpacks"
 weight: 50
 featureId: build.buildpacks
 ---
 
-[Buildpacks]((https://buildpacks.io/)) enable building language-based containers from source code, without the need for a Dockerfile.
+[Cloud Native Buildpacks](https://buildpacks.io/) enable building language-based
+containers from source code, without the need for a Dockerfile.
 
-Skaffold supports building with buildpacks natively.
- 
-Skaffold buildpacks support, builds the image inside local docker daemon. 
-It mounts the source dependencies and local artifact cache if caching is enabled 
-to a container in a docker daemon. These get unmounted once the build process is finished.
+Skaffold supports building with Buildpacks natively, without installing
+additional software. However, Buildpacks require a local Docker daemon
+to be used.
 
-Once all the necessary data is present, Skaffold wil build inside a container in a docker daemon 
-with image specified in  `builderImage` in the `buildpack` config.
+Once all the necessary data is present, Skaffold will build inside a container,
+with the local Docker daemon, with ther builder image specified in `builder`,
+in the `buildpack` config.
 
-On successful build completion, built images will be pushed to the remote registry. You can choose to skip this step.
+On successful build completion, built images will be pushed to the remote registry.
+You can choose to skip this step.
```

Comment:
Is this behaviour really different from other builders?  Its presence here suggests that it is different — I'd delete it if not.

Reply:
The Docker builder uses similar language.  So I'll keep this.